db/migrate/20230120165646_create_post.rb

```diff
@@ -1,12 +1,14 @@
 class CreatePost < ActiveRecord::Migration[7.0]
-  def change
-    create_table :posts do |t|
-      t.string :Title
-      t.string :Text
-      t.integer :CommentsCounter
-      t.integer :LikesCounter
+   def change
+      create_table :posts do |t|
+         t.string :Title
+         t.string :Text
+         t.datetime :CreatedAt
+         t.datetime :UpdatedAt
+         t.integer :CommentsCounter
+         t.integer :LikesCounter
```

Comment:
- [ ] For the correct notation and to avoid future conflicts please 🙏🏽 use snake_case for naming the columns e.g. `likes_counter`